### PR TITLE
Improve navigation sidebar headings

### PR DESCRIPTION
**Pull request by MorrisJobke**
* remove term Nextcloud in headings - it only adds noise
* unify headings to have them grouped by wording (Upgrade ...)
* reduce the length of headings

Before (left) and after (right):

1 combined

-----

2 combined

-----

3 combined

### CODE REVIEW

File: admin_manual/maintenance/backup.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-====================
-Backing up Nextcloud
-====================
+==========
+Backing up
```
Comment (MariusBluem) [inline admin_manual/maintenance/backup.rst]: I think simply **Backup** sounds better 🤔 @MorrisJobke

Comment (MorrisJobke) [inline admin_manual/maintenance/backup.rst]: Done

**Comment by MorrisJobke** [comment]
stable11 697772e6

**Comment by MorrisJobke** [comment]
stable10 4a6b5b7c

**Pull request merged**